### PR TITLE
add zope.deferedimport and zope.hookable

### PR DESCRIPTION
**Pull request by ChrisBarker-NOAA**
Should be like zope.proxy that was recently merged

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by ChrisBarker-NOAA** [comment]
@conda-forge/help-python: I think this is good to go -- only AppVeyor failing

File: recipes/zope.hookable/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  sha256: 0f325838dbac827a1e2ed5d482c1f2656b6844dc96aa098f7727e76395fcd694
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+
+requirements:
+  build:
+    - python
```
Comment (npavlovikj) [inline recipes/zope.hookable/meta.yaml]: Please remove `python` and `pip` from the `build` section - they need to be in `host`, and only the compiler should stay here.

File: recipes/zope.deferredimport/LICENSE.txt [hunk 2]
```diff
@@ -0,0 +1,44 @@
+Zope Public License (ZPL) Version 2.1
```
Comment (npavlovikj) [inline recipes/zope.deferredimport/LICENSE.txt]: The license file is packaged in the pypi tarball, so please remove it from here.

File: recipes/zope.hookable/LICENSE.txt [hunk 3]
```diff
@@ -0,0 +1,44 @@
+Zope Public License (ZPL) Version 2.1
```
Comment (npavlovikj) [inline recipes/zope.hookable/LICENSE.txt]: The license file is packaged in the pypi tarball, so please remove it from here.

**Comment by ChrisBarker-NOAA** [comment]
:-( -- looks like it couldn't install the compiler -- I don't even know where to start with that.

Funny thing is, Appveyor is now working (and seems to be building the Windows packages)..

**Comment by npavlovikj** [comment]
Everything is green now @ChrisBarker-NOAA  :) Thank you for contributing, and happy holidays 🎉

**Pull request merged**